test_scripts/API/AddCommand/SDL_resends_WARNINGS_to_MOB_when_HMI_respond_WARNINGS_to_one_HMI_portions.txt

```diff
@@ -0,0 +1,14 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties
```

Comment:
@vvvakulenko, According to linked question APPLINK-15210, please clarify priority of result codes that should be returned according to this question:
As Example:
Case 1:
UI.SetGlobalProperties(REJECTED)
TTS.SetGlobalProperties(WARNINGS)

Case 2:
UI.SetGlobalProperties(UNSUPPORTED_RESOURCE)
TTS.SetGlobalProperties(WARNINGS)

1. What result code should SDL return to mobile? 
2. Is response of point 1 applicable for all RPCs in scope of this CRQ?